docs/README.md

```diff
@@ -1,11 +1,19 @@
 # Kueue documentation
 
+Welcome to Kueue - Kubernetes native APIs and controller for queueing jobs.
+
 ## Understand Kueue
 
 Learn about Kueue and its fundamental concepts.
 
 [**View concepts**](concepts)
 
+## Quick start
```

Comment:
why do we call it quick start? is there a slow start :)

perhaps name it Setup (just as the reference header the section is pointing to)

Reply:
jejeje good one :P 

lot's of documentation pages have a small "quick start" section. but I agree with you here, will address in a min